### ISSUE TITLE
Shared folder and private folder

### ISSUE THREAD

**Issue by oefterdal** [jupyterhub/jupyterhub#1816]
Is it possible to give each user their own folder for private files and share a folder with all users?

tree structure like this:

/var/notebooks/shared
/var/notebooks/{username}

**Comment by minrk** [comment]
Yes, this would be configuration for your Spawner. If you are using the default Spawner, which uses local system users, the way to do this is to configure your system's default home directory, typically in `/etc/skel`. You can also execute custom commands as part of Spawner initialization, if you need to.
